pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 # Note: Various modules refer to this system as "encoded", not "fourfront".
 name = "encoded"
-version = "5.1.3"
+version = "5.2.0"
```

Comment:
You mean to 6.0.0?

Reply:
I think so.

(And make sure the version prior to the major version is tagged and pushed before you merge so that there is something to compare to later, or even to branch from if a fix to the old series has to be released.)